src/components/Footer.tsx

```diff
@@ -0,0 +1,85 @@
+import cn from 'classnames';
+
+import { FilterType } from '../types';
+
+interface Props {
+  activeTodos: number;
+  completedTodos: number;
+  filter: string;
+  setFilter: (newFilter: FilterType) => void;
+  clearCompleted: () => void;
+}
+
+export const Footer: React.FC<Props> = ({
+  activeTodos,
+  completedTodos,
+  filter,
+  setFilter,
+  clearCompleted,
+}) => {
+  const handleFilterClick = (
+    event: React.MouseEvent<HTMLAnchorElement>,
+    filterType: FilterType,
+  ) => {
+    event.preventDefault();
+    setFilter(filterType);
+  };
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {activeTodos === 1 ? (
+          `${activeTodos} item left`
+        ) : (
+          `${activeTodos} items left`
+        )}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={cn('filter__link', {
+            selected: filter === 'all',
+          })}
+          data-cy="FilterLinkAll"
+          onClick={(event) => handleFilterClick(event, FilterType.ALL)}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={cn('filter__link', {
+            selected: filter === 'active',
+          })}
+          data-cy="FilterLinkActive"
+          onClick={(event) => handleFilterClick(event, FilterType.ACTIVE)}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/completed"
+          className={cn('filter__link', {
+            selected: filter === 'completed',
+          })}
+          data-cy="FilterLinkCompleted"
+          onClick={(event) => handleFilterClick(event, FilterType.COMPLETED)}
+        >
+          Completed
+        </a>
+      </nav>
+
+      {completedTodos && (
```

Comment:
<img width="262" alt="image" src="https://github.com/mate-academy/react_todo-app-with-api/assets/70220667/d45afa67-4754-4334-b3de-7601489cd3c7">
let's fix that